src/main/java/org/spongepowered/common/mixin/core/item/MixinItemFishingRod.java

```diff
@@ -54,10 +54,12 @@ public ItemStack onItemRightClick(ItemStack itemStack, World world, EntityPlayer
             player.fishEntity.handleHookRetraction();
         } else {
             EntityFishHook fishHook = new EntityFishHook(world, player);
-            if (!Sponge.getGame().getEventManager()
+            if (world.isRemote || !Sponge.getGame().getEventManager()
```

Comment:
If i had to choose between those two I would use `BOTH` as describes more precisely what it does.
`DEFAULT` is just the default, but it does not say anything about the real content, you have to clarify this with javadocs. In addition to that from the definition its easier to understand when using `BOTH`

`side() default BOTH`
"Oh its an subjscription to BOTH sides by default."
vs
`side() default DEFAULT`
"I can see that its the default but I don't get what its meaning is."